src/Libraries/Microsoft.Extensions.Diagnostics.Extra/Microsoft.Extensions.Diagnostics.Extra.json

```diff
@@ -26,22 +26,22 @@
         {
           "Member": "const string Microsoft.Extensions.Diagnostics.Enrichment.ApplicationEnricherTags.ApplicationName",
           "Stage": "Stable",
-          "Value": "env_app_name"
+          "Value": "AppName"
         },
         {
           "Member": "const string Microsoft.Extensions.Diagnostics.Enrichment.ApplicationEnricherTags.BuildVersion",
           "Stage": "Stable",
-          "Value": "env_cloud_roleVer"
+          "Value": "CloudRoleVer"
         },
         {
           "Member": "const string Microsoft.Extensions.Diagnostics.Enrichment.ApplicationEnricherTags.DeploymentRing",
           "Stage": "Stable",
-          "Value": "env_cloud_deploymentRing"
+          "Value": "CloudDeploymentRing"
         },
         {
           "Member": "const string Microsoft.Extensions.Diagnostics.Enrichment.ApplicationEnricherTags.EnvironmentName",
           "Stage": "Stable",
-          "Value": "env_cloud_env"
+          "Value": "CloudEnv"
```

Comment:
Changes above are from another PR, aren't they?

Reply:
Yes, but it got discovered when I ran MakeApiBaselines. The static analyzer that checks these baseline files should have reported this change as a breaking change, but it didn't previously. This will need to be fixed.

Reply:
I see. The actual values in https://github.com/dotnet/extensions/blob/release/8.0/src/Libraries/Microsoft.Extensions.Diagnostics.Extra/Enrichment/ApplicationEnricherTags.cs in `release/8.0` branch is the same you have in your changes. Thanks!